_sass/partials/_mixins.scss

```diff
@@ -24,6 +24,12 @@
   align-items: $align;
 }
 
+@mixin flex-direction($direction: row) {
```

Comment:
if we always pass direction (which is clearer when using the code) then I'd consider removing the default